tests/factory_fixtures/ticket.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from models.tickets import TicketModel
+from models.tickets import TicketStatus
+
+@pytest.fixture
+def ticket_attributes():
+    def _ticket_attributes(issue, tenant, assignedUser, sender):
+        return {
+            "issue": issue,
+            "tenant": tenant.id,
+            "assignedUser": assignedUser.id,
+            "sender": sender.id,
+            "status": TicketStatus.New,
+            "urgency": "high"
+        }
+    yield _ticket_attributes
+
+@pytest.fixture
+def create_ticket(ticket_attributes, create_tenant, create_admin_user, create_join_staff):
+    def _create_ticket(issue="Leaky pipe"):
+        tenant = create_tenant()
+        ticket = TicketModel(**ticket_attributes(issue, tenant, create_admin_user(), create_join_staff()))
+        ticket.save_to_db()
+        return ticket
+    yield _create_ticket
```

Comment:
Very nice! Its good practice to create tests for the factories. If something changes in the future that breaks this ticket factory, then every test that depends on this will probably start to fail. Having an additional test failure that will clearly be a different flavor from all the rest that says "I couldn't create a ticket factory" can be helpful.

Reply:
Good call - added a basic one in the integrations folder~